### PR TITLE
Add English translation

### PR DESCRIPTION
**Pull request by jesusjimsa**
This pull request adds English translation to the locale files.

### CODE REVIEW

File: locale/en_US/LC_MESSAGES/en_US.po [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+msgid "video_info"
+msgstr "<b>ID:</b> <code>{0}</code>\n"
+"<b>Title:</b> <code>{1}</code>\n"
+"<b>Description:</b> <code>{2}</code>\n"
+"<b>Keywords:</b> <code>{3}</code>"
+
+
+# common_handlers.py
+msgid "start_command"
+msgstr "Hey. Write @{0} followed by soome text to search for a video. For example: @{0} hello"
```
Comment (Selutario) [inline locale/en_US/LC_MESSAGES/en_US.po]:
```suggestion
msgstr "Hey. Write @{0} followed by some text to search for a video. For example: @{0} hello"
```

File: locale/en_US/LC_MESSAGES/en_US.po [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+msgid "upld_diff_video"
+msgstr "It's different: continue 👍"
+
+msgid "upld_same_video"
+msgstr "It's the same: cancel 👎"
+
+msgid "upld_send_keywords"
+msgstr "Por último, algunas palabras clave que no aparezcan en la transcripción (max {0} caracteres). "
+"P. ej: belen esteban sueño cansado cansada"
```
Comment (Selutario) [inline locale/en_US/LC_MESSAGES/en_US.po]:
```suggestion
```

**Review by Selutario — CHANGES_REQUESTED** [review]
Thanks for the PR! 🎉 Only minor changes are required

File: locale/en_US/LC_MESSAGES/en_US.po (hunk 2 above)
Comment (Selutario) [reply locale/en_US/LC_MESSAGES/en_US.po]: This msgstr is duplicated

File: locale/en_US/LC_MESSAGES/en_US.po (hunk 1 above)
Comment (jesusjimsa) [reply locale/en_US/LC_MESSAGES/en_US.po]: Done!

File: locale/en_US/LC_MESSAGES/en_US.po (hunk 2 above)
Comment (jesusjimsa) [reply locale/en_US/LC_MESSAGES/en_US.po]: Done!

**Pull request merged**